report/5.Conclusion/Discussion.tex

```diff
@@ -1,4 +1,66 @@
-% !TeX root = ../main.tex
-\chapter{Discussion}
-
-\input{5.Conclusion/QualityAssurance.tex}
\ No newline at end of file
+% !TeX root = ../main.tex
+\chapter{Discussion}
+\section{Discussion}
+This chapter reflects upon the choices made in regards to the developed solution and report, as well as how the group delegated work to the individual group members and assured the quality of said work.
+Furthermore the major choices, not already touched upon are discussed in this chapter.

```

Comment:
comma after upon

Reply:
no wait
comma after furthermore
then no comma after choices, and then no other comma either